src/components/scrolling-carousel/index.tsx

```diff
@@ -61,13 +61,11 @@ export const ScrollingCarousel: FunctionComponent<SliderProps> = ({
 	const mouseUp = (_: MouseEvent) => {
 		setIsDown(false);
 		setShowArrow(showArrows());
-		slider.current!.classList.remove(styles.sliding);
```

Comment:
if you remove this line, the element that is targeted for this `mouseUp` event will ultimately trigger the `onClick` event and it will cause `<a>` elements to trigger a navigation while you still sliding. Check [this](https://github.com/Trendyol/react-carousel/blob/cbb06980b66f5bdfaa43b599f98e27570f493158/CHANGELOG.md#carousel-triggerclickon-prop ) feature if it's what you are trying to solve, and it can be implemented for this component too.

Reply:
I added the `triggerClickOn` prop as in carousel component. Could you review this again?